### PR TITLE
docs: add missing browser versions to 1.26 release

### PR DESCRIPTION
**Pull request by yury-s**

### CODE REVIEW

File: docs/src/release-notes-csharp.md [hunk 1]
```diff
@@ -35,6 +35,17 @@ event.
 To align with web specification, the `WaitUntilState.DOMContentLoaded` value only waits for
 the target frame to fire the `'DOMContentLoaded'` event. Use `WaitUntil: WaitUntilState.Load` to wait for all iframes.
 
+## Browser Versions
```
Comment (mxschmitt) [inline docs/src/release-notes-csharp.md:38]:
```suggestion
### Browser Versions
```

and for the other files too.

**Review by mxschmitt — APPROVED** [review]

File: docs/src/release-notes-java.md [hunk 2]
```diff
@@ -33,6 +33,17 @@ event.
 To align with web specification, the `WaitUntilState.DOMCONTENTLOADED` value only waits for
 the target frame to fire the `'DOMContentLoaded'` event. Use `setWaitUntil(WaitUntilState.LOAD)` to wait for all iframes.
 
+## Browser Versions
```
Comment (mxschmitt) [inline docs/src/release-notes-java.md:36]:
```suggestion
### Browser Versions
```

File: docs/src/release-notes-js.md [hunk 3]
```diff
@@ -46,6 +46,17 @@ event.
 To align with web specification, the `'domcontentloaded'` value only waits for
 the target frame to fire the `'DOMContentLoaded'` event. Use `waitUntil: 'load'` to wait for all iframes.
 
+## Browser Versions
```
Comment (mxschmitt) [inline docs/src/release-notes-js.md:49]:
```suggestion
### Browser Versions
```

File: docs/src/release-notes-python.md [hunk 4]
```diff
@@ -33,6 +33,17 @@ event.
 To align with web specification, the `'domcontentloaded'` value only waits for
 the target frame to fire the `'DOMContentLoaded'` event. Use `wait_until="load"` to wait for all iframes.
 
+## Browser Versions
```
Comment (mxschmitt) [inline docs/src/release-notes-python.md:36]:
```suggestion
### Browser Versions
```

**Review by aslushnikov — APPROVED** [review]

**Pull request merged**